app/controllers/items_controller.rb

```diff
@@ -4,9 +4,13 @@ def new
     type = params[:type]
     druid = params[:druid]
     targets = params[:targets]
+    raise 'missing required params' unless type && druid && targets
```

Comment:
to do validation?  the class that is being passed the params isn't activerecord though